### DISCUSSION TITLE
I don't know what's going on

### DISCUSSION THREAD

**Discussion by WHmuse** [gitkraken/vscode-gitlens#2313] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/70631215/199674322-d280054e-6a79-4bc1-a736-ba89862f44ef.png)
I don't know what's going on

**Comment by WHmuse** [comment]
My plugin is very vague

**Comment by d13** [comment, marked as answer]
Your screenshot shows a bug from the 12.2.x version GitLens. Please update to the latest version of GitLens and this should be resolved.
